waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/DefaultMetaStoreClientFactory.java

```diff
@@ -32,7 +32,7 @@
 
 public class DefaultMetaStoreClientFactory implements MetaStoreClientFactory {
 
-  static final Class<?>[] INTERFACES = new Class<?>[] { CloseableThriftHiveMetastoreIface.class };
+  static Class<?>[] INTERFACES = new Class<?>[] { CloseableThriftHiveMetastoreIface.class };
```

Comment:
Was the error here that an array isn't immutable so therefore shouldn't be a constant with all upper case? If so this could just be named `interfaces` rather than `INTERFACES` and then the `final` doesn't really add much. If we really don't want the contents changed then consider using some kind of immutable array or other collection but I'm OK with just renaming.

Reply:
I looked through the warnings on Jenkins and there's no warning about this one. I must have removed the "final" by mistake. Should I rename to `interfaces` anyway?

Reply:
I think just put the final back as the intention is for it to be a constant even if it technically isn't.